packages/common/src/components/utils/Modal/ModalBase.tsx

```diff
@@ -36,7 +33,13 @@ const ModalBase: React.FC<Props> = ({ open, handleClose, title, buttons, content
       <FocusTrap active={open}>
         <div tabIndex={0}>
           <Backdrop className="ic-backdrop iu-lh-body" onClick={handleClose} />
-          <WrapText role="dialog" className="ic-modal" aria-labelledby="dialog-title" aria-modal="true">
+          <div
```

Comment:
Why not pass styles via `className` instead and keep element as style component?

Reply:
I changed it so that a prop desides if its a large modal or not. Let me know if the solutions was too complicated, or can be done in a better way.